### DISCUSSION TITLE
[auth/missing-client-identifier] for real phone number in debug mode

### DISCUSSION THREAD

**Discussion by tyhour** [invertase/react-native-firebase#6994] [General]
For @react-native-firebase/auth can use real phone number for testing like previous version or not?

The Play Integrity API helps you check that you're interacting with your genuine app on a genuine Android device powered by Google Play services. The Play Integrity API has replaced SafetyNet Attestation and Android Device Verification.

It's new for me because I used to work with  Android Device Verification and SafetyNet Attestation, thanks for your answer.

**Comment by mikehardy** [comment]
> auth/missing-client-identifier

That's a specific error message, and you should investigate how to resolve that specifically

> can use real phone number for testing

Yes, you should be able to.

**Reply by tyhour** [reply]
For test phone number it's working fine, but when I use real number it's show error above. I enabled Play Integrity API already. May I know some missing configuration? thanks

**Reply by mikehardy** [reply]
I see promising results at the top of a standard search: https://search.brave.com/search?q=firebase+phone+auth+auth%2Fmissing-client-identifier&source=desktop

**Reply by tyhour** [reply]
Ok I will check again, I try some solution already but I will change from .jsk to keystore instead. I think because of SHA Certificate or I need to change appVerificationDisabledForTesting to False for real number and True for testing. Thanks for help
